yield_plugin/config/parameters.yaml

```diff
@@ -47,11 +47,11 @@ enable_cooperative_behavior: true
 always_accept_mobility_request: true
 # Acceptable number of timesteps to use the latest known mobility request before switching to yield
 # Value type: Desired
-acceptable_passed_timesteps: 3
+acceptable_passed_timesteps: 10
```

Comment:
A whole second seems like a lot, but if it's been working in the field I suppose it's best not to play around with it.

Reply:
back to 3, it is working well with both values.